### DISCUSSION TITLE
How to enable rendering colored .ttf fonts

### DISCUSSION THREAD

**Discussion by appel1** [AvaloniaUI/Avalonia#15159] [Q&A]
I'm probably missing something obvious, but how can you render text using a colored font like [HONK](https://fonts.google.com/specimen/Honk?query=honk) from Google Fonts? 

I've tried loading it as a resource and installing it as a system font on Windows but no success. 

<img width="349" alt="image" src="https://github.com/AvaloniaUI/Avalonia/assets/973314/138ae518-1808-4a83-94aa-b2754c204ad0">

The same .ttf renders correctly in MS Word. 
<img width="232" alt="image" src="https://github.com/AvaloniaUI/Avalonia/assets/973314/ec6a2073-5aec-454d-b11f-ba85b7816959">

And as a regular web page.
<img width="282" alt="image" src="https://github.com/AvaloniaUI/Avalonia/assets/973314/1ccf6e56-1464-460c-af6d-e9a73bbb585b">

```html
<html>
<head>
<style>
@font-face {
  font-family: 'Honk';
  src: url('Honk-Regular.ttf');
}

body {
  font-family: "Honk";
}
</style>
</head>
<body>
Testing testing
</body>
</html>
```

Maybe related to #12469.

**Comment by Gillibald** [comment]
This is probably an issue with the Skia WASM build

**Reply by appel1** [reply]
It behaves the same both on Windows and WASM.
